index.html

```diff
@@ -4116,91 +4076,298 @@ <h3>Parsing content</h3>
 <!-- - - - - - - - - - - - - Security and Privacy - - - - - - - - - - - - - -->
 <section> <h2 id="security">Security and Privacy</h2>
   <p>
-    The trust model, attacker model, threat model and possible mitigation
-    proposals for Web NFC are presented in the
-    <a href="http://w3c.github.io/web-nfc/security-privacy.html">
-    Security and Privacy</a> document. This section presents the chosen
-    security and privacy model through normative requirements to
-    implementations.
+    NFC technology involves multiple levels of security. Payments done with NFC
+    are considered to be at least as secure as with payment cards at hardware
+    level, but the whole software stack needs to be security hardened.
+    However, data transfers involve more threats. In order to mitigate these,
```

Comment:
NDEF based data transfers?

Reply:
NFC data transfers would be better term in a security section. Used that.